homework/index.js

```diff
@@ -1,46 +1,94 @@
 'use strict';
 
-{
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
+async function fetchJson(url) {
```

Comment:
This function doesn't look right, but you don't use it so it can just be removed!

Reply:
Also it gives a syntax error when loading, so that is a problem.

Reply:
You mean we don't need to do a XMLHttpRequest at all?

Reply:
Exactly, you use `fetch` instead :)